lib/typhoeus/response/header.rb

```diff
@@ -47,7 +47,7 @@ def parse
       # @return [ void ]
       def process_line(header)
         key, value = header.split(':', 2)
-        process_pair(key.strip, value.strip.gsub(/\r?\n\s*/, ' '))
+        process_pair(key.strip, (value ? value.strip.gsub(/\r?\n\s*/, ' ') : nil))
```

Comment:
You're welcome ;) as you wish, I have a slight preference for `nil` for the reason explained above, but I asked my coworkers and they have a slight preference for empty string to avoid breaking any code using this hash and expecting it to be all string. I guess if you asked this question you have a slight preference for empty string too? maybe we should go with that then, it's the safer choice.